public/doc/swagger-2.yaml

```diff
@@ -311,16 +311,18 @@ paths:
       parameters:
         - $ref: '#/parameters/PortfolioItemID'
       responses:
-        204:
+        '204':
```

Comment:
@mkanoor - are the quotes a requirement for the swagger.yaml file or is this a new procedure we're all following?